server/zanata-frontend/src/frontend/app/styles/style.less

```diff
@@ -6649,6 +6649,12 @@ table.tmx-table .badge .n1 {
   -moz-hyphens: initial;
   hyphens: initial;
 }
+
+.new-zanata .button--group input[type=radio]:checked+.button--Off {
+  background-color: @color-dark !important;
+  color: #f2f2f2;
```

Comment:
Is this color used for these copyTrans button only? or its a value from our color scheme?